src/firecracker/src/api_server/parsed_request.rs

```diff
@@ -276,7 +276,7 @@ pub(crate) enum RequestError {
     #[error("API Resource IDs can only contain alphanumeric characters and underscores.")]
     InvalidID,
     // The HTTP method & request path combination is not valid.
-    #[error("Invalid request method and/or path: {} {0}.", std::str::from_utf8(.1.raw()).expect("Cannot convert from UTF-8"))]
+    #[error("Invalid request method and/or path: {} {0}.", .1.to_str())]
```

Comment:
Heh, good catch. HTTP protocol itself should be mostly plain ASCII[1], but fixing that is probably out of the scope of this PR

[1]: https://stackoverflow.com/questions/818122/which-encoding-is-used-by-the-http-protocol